hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -231,6 +231,19 @@ getOutputs tx =
   let TxBody TxBodyContent{txOuts} = getTxBody tx
    in txOuts
 
+executionCost :: Ledger.Alonzo.PParams LedgerEra -> CardanoTx -> Lovelace
```

Comment:
There is a `ProtocolParameters` `cardano-api` equivalent in case this aligns better with call sites.

Reply:
Hmm. I purposely avoided that one because I remembered this `ProtocolParameters` to be tricky to convert from. But, I see that `Cardano.Api` does expose `toLedgerPParams` / `fromLedgerPParams` nowadays, so it may be worth the shot.